src/adapters/adapter.js

```diff
@@ -60,6 +60,36 @@ class Adapter {
             item.text = name
             item.icon = type // uses `type` as class name for tree node
 
+            if (item.patch) {
+              let patch_html = ''
+
+              if (item.patch.action) {
+                if (item.patch.action === 'add') {
+                  patch_html += '<span class="text-green">added</span>'
+                }
+                else if (item.patch.action === 'rename') {
+                  patch_html +=
+                    `<span class="text-green"
+                      title="${item.patch.previous}">renamed
+                    </span>`
+                }
+              }
+
+              if (item.patch.filesChanged) {
+                const fileString = item.patch.filesChanged === 1 ? 'file' : 'files'
+                patch_html += `<span>${item.patch.filesChanged} ${fileString}</span>`
+              }
+
+              if (item.patch.additions !== 0 || item.patch.deletions !== 0) {
```

Comment:
For zero number, I think it makes more sense to have it empty. `+0` or `-0` is not informative to me.